subtests/docker_cli/run_volumes/run_volumes.py

```diff
@@ -97,7 +97,9 @@ def init_path_info(self, path_info, host_paths, cntr_paths, tmpdir):
             os.mkdir(host_path)
             # keys must coorespond with those used in *_template strings
             args = self.make_test_files(host_path)
-            args += (host_path, cntr_path)
+            # Inexplicable: pylint 1.7.1 (F26) can't grok '+= (host,cntr)'
+            args += (host_path)
+            args += (cntr_path)
```

Comment:
E1121:102,24: volumes_base.init_path_info: Too many positional arguments for staticmethod call

This one is bizarre. I've tried other variations; this is the only one I hit upon that silenced pylint.